README.md

```diff
@@ -1,2 +1,17 @@
-# thermal_base
-Decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+# Base codes for Thermography
+
+A python package for decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+
+## Install
+Install this package with `pip install git+ssh://git@github.com/detecttechnologies/thermal_base.git@main`
+
+**Disclaimer**
+
+Tested against data from: 
+* Flir encoding format: DJI XT-2, Flir E-40(Temperature values go out of bounds of the image for <255 height)
+* DJI-encoding format: DJI Zenmuse H20-T
+* Thermapp (available in past commits)
+
+## Notes
```

Comment:
Can you add a `## Usage` section above this, with sample code blocks for how this library is to be used after it has been pip-installed?